filebeat/scripts/create_fileset.py

```diff
@@ -46,13 +46,16 @@ def generate_module(base_path, metricbeat_path, module, fileset):
 
     module_path = base_path + "/module/" + module
     meta_path = module_path + "/_meta"
+    kibana_default_path = meta_path + "/kibana/default"
```

Comment:
yes, it's not ideal the export thing, but I am planning to improve it for the next beta release.